gitops/charts/traffic-court-dev-values.yaml

```diff
@@ -18,7 +18,7 @@ citizen-api:
     pullPolicy: Always
   env:
     "FormRecognizer__ApiVersion": "2.1"
-    "FormRecognizer__ModelId": "75f5614c-eded-4413-a3d9-c67281e8402e"
+    "FormRecognizer__ModelId": "498dd4e9-f54c-47ef-be08-5976caf1e07f"
```

Comment:
This is the modelid of the old VT1 model (updated when FOTT moved the model to the new folder structure).
Eventually, this will update to reference the composite model that contains both VT1 and VT2.